utils/setup.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+PREFIX=""
+REGION="${AWS_DEFAULT_REGION}"
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+function title() {
+    echo -e "\033[0;32m$*\033[0m"
+}
+
+usage() {
+    cat <<EOF >&2
+Usage: $0 -p PREFIX [-r REGION]
+  PREFIX: The common bucket prefix. Buckets will be named like "\${PREFIX}-\${FUNCTION}-\${REGION}
+  REGION: The AWS region to use. Defaults to \$AWS_DEFAULT_REGION.
+EOF
+    exit 1
+}
+
+while getopts "p:r:" opt; do
+    case $opt in
+        p)
+            PREFIX=$OPTARG
+            ;;
+        r)
+            REGION=$OPTARG
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            usage
+            ;;
+        :)
+            echo "Option -$OPTARG requires an argument" >&2
+            usage
+            ;;
+    esac
+done
+
+if [ -z "$PREFIX" ]; then
+    echo "Prefix parameter is mandatory" >&2
+    usage
+fi
+
+if [ -z "$REGION" ]; then
+    echo "Please set the AWS region, either using the region argument or \$AWS_DEFAULT_REGION environment variable" >&2
+    usage
+fi
+
+which go >/dev/null
+if [ $? -ne 0 ]; then
+    echo "Go compiler not found. Please install the go compiler." >&2
+    usage
+fi
+
+TMPDIR=`mktemp -d`
+function remove_tmp_dir {
+    rm -rf "${TMPDIR}"
+}
+trap remove_tmp_dir EXIT
+
+export GOPATH="${TMPDIR}"
+export GOOS=linux
+export GOARCH=amd64
+export AWS_DEFAULT_REGION="${REGION}"
+
+title "Creating S3 buckets"
+for func in tile-assets rawr-tiles missing-bucket meta-tiles; do
+   aws s3 mb "s3://${PREFIX}-${func}-${REGION}" --region "${REGION}"
+done
+
+title "Installing Go dependencies"
+mkdir -p "${TMPDIR}/src/tzops"
+ln -sT "${DIR}/../go" "${TMPDIR}/src/tzops/go"
+go get github.com/aws/aws-sdk-go
+go get gopkg.in/yaml.v2
```

Comment:
At some point we should probably rename the imports just to make it `go get`able to avoid having to list these dependencies.